spec/requests/registrations_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Registration', type: :request do
+  describe 'Email registration' do
+    let(:signup_url) { '/auth' }
```

Comment:
```suggestion
    let(:sign_up_url) { '/auth' }
```
Sounds weird `signup` because in the controller you are using `sign_up_params`